app/controllers/companies_controller.rb

```diff
@@ -8,27 +10,22 @@ def new
   end
 
   def create
-    @company = Company.new(company_params)
-    if @company.save
-      flash[:success] = "#{@company.name} added!"
-      redirect_to company_path(@company)
-    else
-      render :new
-    end
+    @company = Company.create!(company_params)
+
+    flash[:success] = "#{@company.name} added!"
```

Comment:
i would leave the if else in here to provide a way to check if things were left out such as title